src/interfaces/IPoolManager.sol

```diff
@@ -91,21 +91,6 @@ interface IPoolManager is IProtocolFees, IERC6909Claims, IExtsload {
     /// @notice Returns the constant representing the minimum tickSpacing for an initialized pool key
     function MIN_TICK_SPACING() external view returns (int24);
 
-    /// @notice Get the current value in slot0 of the given pool
-    function getSlot0(PoolId id)
-        external
-        view
-        returns (uint160 sqrtPriceX96, int24 tick, uint24 protocolFee, uint24 lpFee);
-
-    /// @notice Get the current value of liquidity of the given pool
-    function getLiquidity(PoolId id) external view returns (uint128 liquidity);
-
-    /// @notice Get the current value of liquidity for the specified pool and position
-    function getLiquidity(PoolId id, address owner, int24 tickLower, int24 tickUpper, bytes32 salt)
-        external
-        view
-        returns (uint128 liquidity);
-
     /// @notice Getter for TickInfo for the given poolId and tick
     function getPoolTickInfo(PoolId id, int24 tick) external view returns (Pool.TickInfo memory);
```

Comment:
are we able to remove `getPoolTickInfo` `getFeeGrowthGlobals` `getPoolBitmapInfo` too?

Reply:
Yeah theres quite a bit of functions that can be removed and added to a new library, but I think we should resolve them in a separate PR.

Reply:
ie also in ProtocolFees.sol we could just use extsload

Reply:
> getFeeGrowthGlobals
Looks like you did already add this (just without the `s`. So can remove this from the pool manager!
> getPoolBitmapInfo
You also already added this named `getTickBitmap` so you can remove this from the pool manager too